jdaviz/app.py

```diff
@@ -1796,6 +1798,14 @@ def _reparent_subsets(self, old_parent, new_parent=None):
                     # Translate bounds through WCS if needed
                     if (self.config == "imviz" and
                             self._jdaviz_helper.plugins["Orientation"].link_type == "WCS"):
+
+                        old_angle, _,  old_flip = get_compass_info(old_parent.coords, (10, 10))[-3:]
+                        new_angle, _, new_flip = get_compass_info(new_parent.coords, (10, 10))[-3:]
+                        relative_angle = new_angle - old_angle
+                        print(old_angle, new_angle, relative_angle)
```

Comment:
```suggestion
```

Reply:
Yes thanks, there's a few things to do, that's why it's in draft.